verifyFiles.py

```diff
@@ -345,11 +359,11 @@ def main():
         else:
 	    log_info("Pretty printing...")
             print pretty_string(diff,conditions_dict)
-        for metric_name in metric_values.keys():
-            log_info("Writing values of metric \""+metric_name+"\" to file \""+metrics[metric_name]["output_file"]+"\"")
-            metric_file = open(metrics[metric_name]["output_file"],'w')
-	    metric_file.write(pretty_string(metric_values[metric_name],conditions_dict))
-	    metric_file.close()
+        #for metric_name in metric_values.keys():
+            #log_info("Writing values of metric \""+metric_name+"\" to file \""+metrics[metric_name]["output_file"]+"\"")
+            #metric_file = open(metrics[metric_name]["output_file"],'w')
+	    #metric_file.write(pretty_string(metric_values[metric_name],conditions_dict))
+	    #metric_file.close()
```

Comment:
I knew that the code needed improvement. Commit came into the pull request as it was pending. Will make the changes requested and let you know.